src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -48,8 +48,15 @@ struct PathManagerParams
         internal IPathResolver PathResolver { get; set; }
     }
 
-    class PathManager : IPathManager
+    public sealed class PathManager : IPathManager
     {
+        internal static Lazy<PathManager>
+         lazy =
+         new Lazy<PathManager>
```

Comment:
consider everytime that you make something a singleton it's further from ever being able to run two dynamo models in the same process.. I guess we're giving up on that :).

Reply:
> 

The two classes that are being converted to singleton are like helpers for the DynamoModel, In which scenario will be required two instances of DynamoModel in the same process ?, Maybe I'm missing something.